rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+*** *Plain Text (.txt)*
+*** *Web Page (HTML) (.zip)*
+*** *EPUB (.epub)*
++
+** *Sheets Type*
++
+File format for exporting Google Sheets. Available types:
++
+*** *Microsoft Excel (.xlsx)*
+*** *OpenDocument (.ods)*
+*** *PDF (.pdf)*
+*** *Web Page (HTML) (.zip)*
+*** *Comma-Separated Values (first-sheet only) (.csv)*
+*** *Tab-Separated Values (first-sheet only) (.tsv)*
++
+** *Slides Type*
++
+File format for exporting Google Slides. Available types:
++
+*** *Microsoft PowerPoint (.pptx)*
+*** *ODP (.odp)* 
+*** *PDF (.pdf)* 
+*** *Plain Text (.txt)*
+*** *JPEG (first-slide only) (.jpg)*
+*** *PNG (first-slide only) (.png)*
+*** *Scalable Vector Graphics (first-slide only) (.svg)*
+
+* *Google Drive File Settings*
+** *Create directory if it does not exist*
++
+When selected, creates the folder specified in *Directory path* if it does not exist.
+** *Directory path* 
++
+Path to the directory where to download or export the file. 
+** *New file name* 
++
+Filename to use when *Set new file name* is enabled. You must specify the file name including the extension. 
+** *Overwrite existing file* 
++
+When selected, overwrites any existing file in the specified download directory that has the same name as the downloaded or exported file.
+** *Set new file name*
++
+When selected, renames the downloaded or exported file using the specified *New file name*.
+
+== Inbound Variables 
+
+* *File ID*
+* *Directory Path* 
+* *Create directory if it does not exist*
+* *Overwrite existing file* 
+* *Set new file name* 
+* *New file name* 
+* *Credential Type*
+* *Project ID* 
+* *Private Key ID* 
+* *Private Key* 
+* *Token URI* 
+* *Auth Provider x509 Cert URL* 
+* *Auth URI* 
+* *Client Email* 
+* *Client ID* 
+* *Client x509 Cert URL* 
+* *Universe Domain* 
+* *Google API Service Account Credentials* 
+
+== Outbound Variables 
+
+* *Download Succeeded* (Boolean)
++
+Confirms if the download was successful. 
+* *Target folder* 
++
+String containing the directory path to the downloaded or exported file. 
+* *File name* 
++
+String containing the file name, including the extension. 
+
+== Wizard 
+
+Specify your authentication settings for the Google API service account in the *Authentication Settings* panel. Select *Pin credentials* to specify the activity parameter containing the credentials, or select *Use custom settings* to provide the information inline. You can either import a JSON file with your Google API credentials information or specify the authentication information manually. 
+
+In *Download Settings*, select between *Download file* or *Export Google Workspace document*, depending on the operation to perform. When exporting a file, you must select the type of Google file to export (Docs, Sheets, or Slides) and the format for the target file. Available export formats depend on the file being exported. 
+
+Provide the *File ID* of the file to download or export. To obtain this ID, use the Search in Google Drive action step. 
```

Comment:
Our style says not to use UI elements in this way. Suggest lowercase "file ID" and no bold.

Should the action step be init cap and bold? I think Simone does this. See this [example](https://github.com/mulesoft/docs-rpa/pull/296/files#:~:text=You%20use%20the%20*Search%20Pattern*%20Action%20Step%20to%20search%20for%20this%20arrow%20up%20symbol%20and%20limit%20the%20search%20area%20to%20the%20space%20directly%20below%20the%20column%20name.).